code/game/objects/structures/watercloset.dm

```diff
@@ -381,7 +380,7 @@
 			else if(H.w_uniform && wash_obj(H.w_uniform))
 				H.update_inv_w_uniform()
 			if(washgloves)
-				SEND_SIGNAL(H, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)
```

Comment:
Ditto. There's plenty of other instances where you removed the clean_act signal. Readd 'em. You don't have to replace clean_blood() to do it.